javascript/src/defineDefaultParameterTypes.ts

```diff
@@ -60,7 +60,7 @@ export default function defineDefaultParameterTypes(registry: DefinesParameterTy
     new ParameterType(
       'bigdecimal',
       FLOAT_REGEXP,
-      Number,
+      String,
```

Comment:
?

Reply:
There is no native BigDecimal for JavaScript so it will not be transformed

Reply:
So why not Number?

Reply:
Because the precision of Number is inferior to a BigDecimal

Reply:
That makes it a breaking change then?

Reply:
> • Fixed the type on a couple of built-in parameter types

The other one is a breaking change too. Though I dont' think it will be used much.

Reply:
It's not a breaking change because the `type` field was previously private (and therefore not used by any clients)

Reply:
Gotcha.